homeassistant/const.py

```diff
@@ -122,6 +123,7 @@
 CONF_QUOTE = 'quote'
 CONF_RADIUS = 'radius'
 CONF_RECIPIENT = 'recipient'
+CONF_REFRESH_TOKEN = 'refresh_token'
```

Comment:
Please keep these constants local to the integration.

Reply:
I think that a `refresh_token`, being part of OAuth specification, would be OK for this. Same for `access_token_expires`?

I will take them out now, but will happily put them back if you change your mind.